### PR TITLE
New gpg credential validator

### PR DESCRIPTION
**Pull request by jameswinkler**

### CODE REVIEW

File: Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Internal
```
Comment (eddynaka) [inline Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs:8]: >Internal [](start = 71, length = 8)

remove internal

Comment (jameswinkler) [reply Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs]: Done.

---
In reply to: [585942246](https://github.com/microsoft/sarif-pattern-matcher/pull/249#discussion_r585942246) [](ancestors = 585942246)

File: Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            return IsValidStatic(Instance,
+                                 ref matchedPattern,
+                                 ref groups,
+                                 ref failureLevel,
+                                 ref fingerprint,
+                                 ref message);
+        }
+
+        public static string IsValidDynamic(ref string fingerprint, ref string message)
```
Comment (eddynaka) [inline Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs:33]: remove this

Comment (jameswinkler) [reply Src/Plugins/Security/SEC101_051.GpgCredentialValidator.cs]: Done

---
In reply to: [585980602](https://github.com/microsoft/sarif-pattern-matcher/pull/249#discussion_r585980602) [](ancestors = 585980602)

**Review by eddynaka — APPROVED** [review]

**Pull request merged**